### ISSUE TITLE
Merge mrm into JPMassar

### ISSUE THREAD

**Issue by ctm** [ctm/mb2-doc#486]
Due to ambiguity / confusion associated with various *ARG* sign-up forms, JP has some of his games as mrm and some as JPMassar. He'd like them all to be under JPMassar although I believe it's a low priority for him.

I should really bite the bullet and start using stored procedures for a bunch of administrative stuff, since that's how the administrative console will work anyway and I can crank out the stored procedures w/o any concern about UI; it's just a question of arguments and semantics.

So... might want to start here. As such, I'm marking this as high priority, not because this particular issue is timely, but because it opens a new avenue for development and can be something I work on when I need my brain to think in. different diredtion.

**Comment by ctm** [comment]
So, I made this `high priority` on December 18th, but haven't done anything directly with it since. I'm about to strip that tag from a bunch of issues, but this one goes hand in hand with using stored procedures (#516), so I'll probably do that one and this one at the same time, "soon".

**Comment by ctm** [comment]
Done. Turns out, I thought I was going to do this with a stored procedure, but I decided against it. Instead I chose to use Rust macros so that I could have a macro that iterated over various fields and "did the right thing". This took me a while, but I now have a `db_tool` application that has a `merge` sub-command that does the right thing.

Although it took longer to do it this way, I think it's much more robust than a stored procedure, because each time I do a build, Rust's type system does a hell of a lot more sanity checking than a stored procedure does.

Whee!
